src/containers/Subscriptions.js

```diff
@@ -52,7 +52,7 @@ export const Subscriptions = props => {
               background: 'rbg(255, 255, 255, 0.3)'
             })
           }}
-          text={<RingLoader sizeUnit={'px'} size={200} color={'#ee7335'} />}
+          text={<CustomRingLoader />}
```

Comment:
could we maybe have color as a prop, so that we can keep this color different from the other ones? 
If I remember correctly, I made it the AV orange color because the AV blue is difficult to see... or maybe you could take a screenshot of this one to see how it looks?